### PR TITLE
Fixing the label size and cleaning up the spacing on the other fields

### PR DESCRIPTION
**Pull request by erikajohnson**

### CODE REVIEW

File: src/Fields/identity.tsx [hunk 1]
```diff
@@ -83,7 +83,9 @@ export default props => {
                 <div className="popover-top">
                   <div className="popover-header flex-center flex-row">
                     <div className="flex-grow">
-                      <Label className="font-weight-semibold">Assign to</Label>
+                      <div className="font-weight-semibold body-m">
+                        Assigned to
```
Comment (cschleiden) [inline src/Fields/identity.tsx]: `To`?

Comment (erikajohnson) [reply src/Fields/identity.tsx]: sentence case?

File: src/Fields/state.tsx [hunk 2]
```diff
@@ -54,9 +54,7 @@ export default props => {
                 <div className="popover-top">
                   <div className="popover-header flex-center flex-row">
                     <div className="flex-grow">
-                      <Label className="font-weight-semibold body-m">
-                        State
-                      </Label>
+                      <div className="font-weight-semibold body-m">State</div>
```
Comment (cschleiden) [inline src/Fields/state.tsx:57]: Let's keep the Label component

File: src/Fields/identity.tsx (hunk 1 above)
Comment (cschleiden) [reply src/Fields/identity.tsx]: But now line 67 and 87 in conflict

File: src/Fields/state.tsx (hunk 2 above)
Comment (erikajohnson) [reply src/Fields/state.tsx:57]: then the Label component needs to change to not specify body-s inside it

Comment (erikajohnson) [reply src/Fields/state.tsx:57]: and all instantiations of Label will have to specify their font size

File: src/Fields/identity.tsx (hunk 1 above)
Comment (erikajohnson) [reply src/Fields/identity.tsx]: oh. they should both be sentence case

**Pull request closed without merging**